python/sage_helper.py

```diff
@@ -22,11 +22,73 @@ def sage_method(function):
         function._sage_method = True
         return function
 
+    try:
+        # Monolithic Sage library
+        from sage.all import RealField, RealDoubleElement, gcd, xgcd, prod, powerset
+        from sage.all import MatrixSpace, matrix, vector, ZZ
+        from sage.all import Integer, Rational, QQ, RR, CC
+        from sage.all import sqrt
+        from sage.all import I, Infinity
+        from sage.all import arccosh
+        from sage.all import RIF, CIF
```

Comment:
Do we actually need this?

Note: I do use the name RIF and CIF for RealIntervalField and ComplexIntervalField instances with precisions different from RealIntervalField() and ComplexIntervalField().